app/helpers/application_helper.rb

```diff
@@ -423,11 +423,11 @@ def current_host_details_path(host)
     Setting['host_details_ui'] ? host_details_page_path(host) : host_path(host)
   end
 
-  def hosts_path(*args)
+  def current_hosts_path(*args)
```

Comment:
Nit: is `current` the best naming? Perhaps `preferred` is better, though it's also not great either.

Reply:
This is consistent with `current_host_details_path` which was already there. :shrug: